tools/bigquery-zos-mainframe-connector/USING_WITH_BQ.md

```diff
@@ -0,0 +1,227 @@
+# Using with BigQuery
+
```

Comment:
add note about using external table vs native table: if the dataset is unchanged after load, it makes sense to use a native table; if there are transformations, it is better to register as external then apply the transformations on the external table so that the data is only scanned once.

Reply:
added